documentation/import-production-data-local.md

```diff
@@ -0,0 +1,88 @@
+# Import production data from cloud firestore to the local firestore
+
+based on this [tutorial](https://medium.com/firebase-developers/how-to-import-production-data-from-cloud-firestore-to-the-local-emulator-e82ae1c6ed8) with a few differences for our use case
+
+The newest version of the OKR-Tracker uses the Firebase Local Emulator Suite, where you can play and test your data without beging afraid of production changes. It is still in the early stages, which means that auth is still handled by the cloud firebase and not locally.
+
+When you start up the local firestor emulator you can see that the firestor is completely empty because we don't have any production data. This is an amazing way of working because you can do what ever you want without doing damages, but it's real life data that you most likely want to test and fix.
```

Comment:
When you start up the local Firestore emulator you can see that the Firestore is completely empty because we don't have any production data. This is an amazing way of working because you can do what ever you want without doing damages, but it's real life data that you most likely want to test and fix.